doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -284,155 +294,81 @@ Induction
+    (see :ref:`bindings`).
 
-   Allows to give explicit instances to the premises of the type of :n:`@term`
-   (see :ref:`bindings`).
+  :n:`{? using @one_term {? with @bindings } }`
+    Allows explicitly giving an induction principle :n:`@one_term` that
+    is not the standard one for the underlying inductive type of :n:`@one_term`. The
+    :n:`@bindings` clause allows instantiating premises of the type of
+    :n:`@one_term`.
 
-.. tacv:: eelim @term
-   :name: eelim
+.. tacn:: eelim @one_term_with_bindings {? using @one_term {? with @bindings } }
 
-   In case the type of :n:`@term` has dependent premises, this turns them into
+   If the type of :n:`@one_term` has dependent premises, this turns them into
    existential variables to be resolved later on.
 
-.. tacv:: elim @term using @term
-          elim @term using @term with @bindings
-
-   Allows the user to give explicitly an induction principle :n:`@term` that
-   is not the standard one for the underlying inductive type of :n:`@term`. The
-   :n:`@bindings` clause allows instantiating premises of the type of
-   :n:`@term`.
-
-.. tacv:: elim @term with @bindings using @term with @bindings
-          eelim @term with @bindings using @term with @bindings
-
-   These are the most general forms of :tacn:`elim` and :tacn:`eelim`. It combines the
-   effects of the ``using`` clause and of the two uses of the ``with`` clause.
-
-.. tacv:: elimtype @type
-   :name: elimtype
+.. tacn:: elimtype @one_term
 
    The argument :token:`type` must be inductively defined. :n:`elimtype I` is
-   equivalent to :n:`cut I. intro Hn; elim Hn; clear Hn.` Therefore the
+   equivalent to :tacn:`cut` :n:`I. intro Hn; elim Hn;` :tacn:`clear` :n:`Hn.` Therefore the
    hypothesis :g:`Hn` will not appear in the context(s) of the subgoal(s).
-   Conversely, if :g:`t` is a :n:`@term` of (inductive) type :g:`I` that does
+   Conversely, if :g:`t` is a :n:`@one_term` of (inductive) type :g:`I` that does
    not occur in the goal, then :n:`elim t` is equivalent to
-   :n:`elimtype I; 2:exact t.`
+   :n:`elimtype I; 2:` :tacn:`exact` `t.`
 
-.. tacv:: simple induction @ident
-   :name: simple induction
+.. tacn:: simple induction {| @ident | @natural }
 
-   This tactic behaves as :n:`intros until @ident; elim @ident` when
+   Behaves like :n:`intros until {| @ident | @natural }; elim @ident` when
    :n:`@ident` is a quantified variable of the goal.
 
-.. tacv:: simple induction @natural
+.. tacn:: dependent induction @ident {? {| generalizing | in } {+ @ident } } {? using @one_term }
 
-   This tactic behaves as :n:`intros until @natural; elim @ident` where :n:`@ident`
-   is the name given by :n:`intros until @natural` to the :n:`@natural`-th non-dependent
-   premise of the goal.
-
-.. tacn:: dependent induction @ident
-   :name: dependent induction
-
-   The *experimental* tactic dependent induction performs induction-
+   The *experimental* tactic :tacn:`dependent induction` performs induction-
```

Comment:
There shouldn't be a newline between `induction-` and `inversion`. It introduces a spurious space.